scripts/Create_table_mh_thresholds.sql

```diff
@@ -1,14 +1,25 @@
 -- create lookup table for multi hazard thresholds value to be used
 DROP TABLE IF EXISTS mh.mh_thresholds CASCADE;
 
-SELECT
-0.028 AS "MHInt_t",
-0.115 AS "PGAt",
-0.05 AS "MMI7t",
-20 AS "Tsun_t",
-30 AS "Fl500t",
-3.1 AS "LndSust",
-2000 AS "Firet",
-177 AS "Cy500t"
+-- create table
+CREATE TABLE mh.mh_thresholds(
+    PRIMARY KEY(threat),
+    threat varchar,
+    htt_exposure float,
+    hti_pgv500 float,
+    hti_pga500 float,
+    hti_tsun500 float,
+    hti_fld500 float,
+    hti_wildfire float,
+    hti_lndsus float,
+    hti_cy500 float
+);
 
-INTO mh.mh_thresholds;
\ No newline at end of file
+/*
+-- import exposure from csv
+COPY mh.mh_thresholds(threat,htt_exposure,hti_pgv500,hti_pga500,hti_tsun500,hti_fld500,hti_wildfire,hti_lndsus,hti_cy500)
+    FROM 'D:\Workspace\data\source datasets\mh\hazard-threat_thresholds.csv'
```

Comment:
Might need to change `D:\Workspace\data\source datasets\mh\` to `/usr/src/app/` or something like that?

Reply:
Will remove